flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/test/program/TableTestStep.java

```diff
@@ -151,5 +153,30 @@ public SpecificBuilder addPartitionKeys(String... partitionKeys) {
             this.partitionKeys.addAll(Arrays.asList(partitionKeys));
             return (SpecificBuilder) this;
         }
+
+        public SpecificBuilder enforceAppendOnly() {
+            this.options.put("connector", "values");
+            this.options.put("changelog-mode", "I");
+            this.options.put("sink-changelog-mode-enforced", "I");
+            return (SpecificBuilder) this;
+        }
+
+        public SpecificBuilder inMode(ChangelogMode mode) {
```

Comment:
```suggestion
        public SpecificBuilder addMode(ChangelogMode mode) {
```
Sorry for the back and forth, but it seems `add` better fits to `addOption` and `addSchema`